src/SqlPersistence/Config/SqlPersistenceConfig_SqlDialect_MsSqlServer.cs

```diff
@@ -11,8 +11,16 @@ public static void Schema(this SqlDialectSettings<SqlDialect.MsSqlServer> dialec
             Guard.AgainstNull(nameof(dialectSettings), dialectSettings);
             Guard.AgainstNullAndEmpty(nameof(schema), schema);
             Guard.AgainstSqlDelimiters(nameof(schema), schema);
-            dialectSettings.Settings.Schema = schema;
+            dialectSettings.TypedDialect.Schema = schema;
         }
 
+        /// <summary>
+        /// Instructs the persistence to not use the connection established by the SQL Server transport.
```

Comment:
In that case, we also can't detect when they have it misconfigured and throw an appropriate exception, because we don't even try creating a connection until we've determined we can't enlist, right?

I'm not sure I like that very much. I wouldn't want to wait around for an "invalid object name" exception with no further clue to what's really going on.